src/main/java/com/dream11/aerospike/client/AerospikeClient.java

```diff
@@ -1,47 +1,60 @@
 package com.dream11.aerospike.client;
 
-import com.aerospike.client.*;
+import com.aerospike.client.AerospikeException;
+import com.aerospike.client.BatchRead;
+import com.aerospike.client.Bin;
+import com.aerospike.client.Key;
+import com.aerospike.client.Operation;
+import com.aerospike.client.Record;
+import com.aerospike.client.Value;
 import com.aerospike.client.cluster.ClusterStats;
-import com.aerospike.client.policy.*;
+import com.aerospike.client.policy.BatchPolicy;
+import com.aerospike.client.policy.Policy;
+import com.aerospike.client.policy.QueryPolicy;
+import com.aerospike.client.policy.ScanPolicy;
+import com.aerospike.client.policy.WritePolicy;
 import com.aerospike.client.query.KeyRecord;
 import com.aerospike.client.query.PartitionFilter;
 import com.aerospike.client.query.Statement;
 import com.dream11.aerospike.config.AerospikeConnectOptions;
-import com.dream11.aerospike.factory.AerospikeClientFactoryImpl;
+import com.dream11.aerospike.util.SharedDataUtils;
 import io.vertx.codegen.annotations.GenIgnore;
 import io.vertx.codegen.annotations.VertxGen;
 import io.vertx.core.AsyncResult;
 import io.vertx.core.Handler;
 import io.vertx.core.Vertx;
-
 import java.util.List;
 
 @VertxGen
 public interface AerospikeClient extends AutoCloseable {
 
   @GenIgnore(GenIgnore.PERMITTED_TYPE)
-  public static AerospikeClient create(Vertx vertx, AerospikeConnectOptions config) {
-    return new AerospikeClientFactoryImpl().getClient(vertx, config);
+  static AerospikeClient createShared(Vertx vertx, AerospikeConnectOptions connectOptions) {
```

Comment:
The create method should create shared Instance as that should be the default behaviour. For non-shared instance, we can use the method name createNonShared